client/src/components/DrawingScreen/DrawingScreen.jsx

```diff
@@ -75,7 +75,7 @@ class DrawingScreen extends React.Component {
   }
 
   setupCanvas(p5, canvasParentRef) {
-    const canvasContainer = p5.createCanvas(500, 500).parent(canvasParentRef);
+    const canvasContainer = p5.createCanvas(900, 900).parent(canvasParentRef);
```

Comment:
is there another way to do this besides hard coding a size? can we have different values per environment?

100% have no idea what is the canonical way to do this stuff so my questions are probably dumb

Reply:
That's actually a great question... https://p5js.org/reference/#/p5/createCanvas seems to take a hardcoded size but maybe there's a way to get that dynamically? It might also be weird if different people's images are different resolutions

Reply:
on my hd monitor this thing looks yuuuge (mebe too yuuuuge)

![image](https://user-images.githubusercontent.com/986174/82129916-689b4800-977b-11ea-95af-3620ee5acd44.png)

Reply:
thats the entire max-window chrome probably we want something a wee bit smaller